README.md

```diff
@@ -28,8 +28,8 @@ Beekeeper comprises two separate Spring-based Java applications. One application
 2. An operation is executed on the table that orphans some data (alter partition, drop partition, etc.)
 3. Hive Metastore events are emitted by the [Hive Metastore Listener](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-listener) as a result of the operation.
 4. Hive events are picked up from the queue by Beekeeper using [Apiary Receiver](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-receivers).
-5. Beekeeper processes these messages and schedules orphaned paths for deletion by adding them to a database.
-6. The scheduled paths are deleted by Beekeeper after a configurable delay, the default is 3 days (see [Hive table configuration](#hive-table-configuration) for more details.)
+5. Beekeeper processes these messages and schedules orphaned paths or expired tables for deletion by adding them to a database.
+6. The scheduled paths or tables are deleted by Beekeeper after a configurable delay, the default is 3 days for tables and 30 days for tables (see [Hive table configuration](#hive-table-configuration) for more details.)
```

Comment:
`3 days for tables and 30 days for tables` ?

Reply:
Whoops I mean't to write paths